ansible/tasks/stickypedia.yml

```diff
@@ -1,76 +1,43 @@
 ---
-- name: "create stickypedia user"
+- name: "stop stickypedia service"
+  systemd:
+    unit: "stickypedia.service"
+    state: "stopped"
+
+- name: "remove stickypedia user"
   user:
     name: "stickypedia"
     shell: "/usr/sbin/nologin"
     home: "/var/www/stickypedia"
     system: true
+    state: "absent"
 
-- name: "install prerequisites for gollum"
+- name: "remove prerequisites for gollum"
```

Comment:
What happens to the packages that are removed from this list? I know that this means that ansible does not care about them, but is that what we want?

Reply:
Those packages are either required for an other role, or `make`, `cmake` and similar packages that cannot be removed